2022-01-26 07:20:01,423 INFO    MainThread:6591 [wandb_setup.py:_flush():69] setting env: {}
2022-01-26 07:20:01,424 INFO    MainThread:6591 [wandb_setup.py:_flush():69] setting login settings: {}
2022-01-26 07:20:01,424 INFO    MainThread:6591 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220126_072001-grandpp_test/logs/debug.log
2022-01-26 07:20:01,424 INFO    MainThread:6591 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220126_072001-grandpp_test/logs/debug-internal.log
2022-01-26 07:20:01,425 INFO    MainThread:6591 [wandb_init.py:init():369] calling init triggers
2022-01-26 07:20:01,425 INFO    MainThread:6591 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-01-26 07:20:01,425 INFO    MainThread:6591 [wandb_init.py:init():418] starting backend
2022-01-26 07:20:01,425 INFO    MainThread:6591 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-26 07:20:01,475 INFO    MainThread:6591 [backend.py:ensure_launched():123] starting backend process...
2022-01-26 07:20:01,484 INFO    MainThread:6591 [backend.py:ensure_launched():128] started backend process with pid: 6641
2022-01-26 07:20:01,485 INFO    MainThread:6591 [wandb_init.py:init():423] backend started and connected
2022-01-26 07:20:01,486 INFO    MainThread:6591 [wandb_init.py:init():465] updated telemetry
2022-01-26 07:20:01,486 INFO    MainThread:6591 [wandb_init.py:init():484] communicating current version
2022-01-26 07:20:02,221 INFO    MainThread:6591 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-26 07:20:02,221 INFO    MainThread:6591 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-01-26 07:20:02,724 INFO    MainThread:6591 [wandb_init.py:init():522] starting run threads in backend
2022-01-26 07:20:06,263 INFO    MainThread:6591 [wandb_run.py:_console_start():1524] atexit reg
2022-01-26 07:20:06,263 INFO    MainThread:6591 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-01-26 07:20:06,264 INFO    MainThread:6591 [wandb_run.py:_redirect():1376] Redirecting console.
2022-01-26 07:20:06,265 INFO    MainThread:6591 [wandb_run.py:_redirect():1438] Redirects installed.
2022-01-26 07:20:06,265 INFO    MainThread:6591 [wandb_init.py:init():546] run started, returning control to user process
2022-01-26 07:20:16,559 INFO    MainThread:6591 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-01-26 07:20:16,559 INFO    MainThread:6591 [wandb_run.py:_restore():1466] restore
2022-01-26 07:20:20,177 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 988
  total_bytes: 988
}

2022-01-26 07:20:20,748 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 988
  total_bytes: 1020
}

2022-01-26 07:20:21,508 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7016
  total_bytes: 7016
}

2022-01-26 07:20:21,610 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7016
  total_bytes: 7016
}

2022-01-26 07:20:21,713 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7016
  total_bytes: 7016
}

2022-01-26 07:20:21,816 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7016
  total_bytes: 7016
}

2022-01-26 07:20:21,919 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7016
  total_bytes: 7016
}

2022-01-26 07:20:22,023 INFO    MainThread:6591 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7016
  total_bytes: 7016
}

2022-01-26 07:20:23,215 INFO    MainThread:6591 [wandb_run.py:_show_files():1838] logging synced files
